src/_shared/scss/_adverts-jobs.scss

```diff
@@ -49,6 +55,10 @@
             background: darken($news-main-1, 5%);
         }
     }
+
+    @include mq($until: tablet) {
```

Comment:
Yeah you're probably right. The reason I did it that way is that it generates less CSS.